### DISCUSSION TITLE
Consecutive comment lines are not foldable in Python but are in (at least some) other languages

### DISCUSSION THREAD

**Discussion by jolsen-mitsu** [microsoft/pylance-release#5905] [Enhancement]
<h3> Environment data </h3>

-   Language Server version: 2024.5.101
-   OS and version: linux x64
-   Python version (and distribution if applicable, e.g. Anaconda):  Python 3.11.9 (main, Apr  6 2024, 17:59:24) [GCC 11.4.0]
-   python.analysis.indexing: true
-   python.analysis.typeCheckingMode: standard
Version: 1.89.1 (user setup)
Commit: dc96b837cf6bb4af9cd736aa3af08cf8279f7685
Date: 2024-05-07T05:13:33.891Z
Electron: 28.2.8
ElectronBuildId: 27744544
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Windows_NT x64 10.0.22621

Only enabled extensions I have, to make sure other extension not interfering with this functionality:
* Pylance 2924,5,191
* Pylint 2023.10.1 preview
* Python 2024.7.11371014

<h3> Code Snippet </h3>

```python
"""'
this will fold
properly
"""

# the folding option
# is missing for these consecutive 
# comments. It works in other languages, 
# like in Javascript with // comment markers
```

![code-folding](https://github.com/microsoft/pylance-release/assets/110049748/1702225b-4559-44b7-8fe5-a27d027a296f)

<h3> Repro Steps </h3>

Add several contiguous single-line comments in python language-selected editor. Hover mouse on the first comment between the comment and the line number column. Ideally, a fold/unfold icon would appear and be actionable. 

<h3> Expected behavior </h3>

I expect the multiple single-line comments to be foldable in Python, like they are in JavaScript

<h3> Actual behavior </h3>

They are not treated as foldable, so there is no option to fold them

<h3> Logs </h3>

After setting logLevel to Trace and restarting VS Code, there is no output in the panel to copy.

**Comment by debonte** [comment]
Moving this issue to discussions as an enhancement request for comments and upvotes.
